doc/release-notes/5.3-release-notes.md

```diff
@@ -0,0 +1,152 @@
+Previously, the configuration of the database connections has been quite static and not very easy to update. This has been an issue especially for cloud and container usage. Using new technologies provided by the move to Payara, you can now more easily configure the connection to your PostgreSQL DB.
+
+Using MicroProfile Config API (Issue #7000, Issue #7418), you can much more easily specify configuration
+details. For an overview of supported options, please see the
+[Installation Guide](https://guides.dataverse.org/en/5.3/installation/config.html#jvm-options).
+
+Note that some settings have been moved from domain.xml to code, such as min and max pool size.
+
+## Major Use Cases
+
+Newly-supported use cases in this release include:
+
+- Users can use an API to add auxiliary files to files in order to provide metadata representations for specific tools or integrations (Issue #7275, PR #7350)
+- Administrators can use a new API to manage banner messages and take advantage of new banner display options (Issue #7263, PR #7434)
+- Users replacing files will now have their files renamed when a file name conflict exists, making the behavior consistent with upload and edit (Issue #7335, PR #7336)
+- Users will now be able to search on file tags on the advanced search and dataset pages (Issue #7194, PR #7385)
+
+## Notes for Dataverse Installation Administrators
+
+### Payara 5.2020.6 Required
+
+Some changes in this release require an upgrade to Payara 5.2020.6.
+
+Instructions how to update can be found in the
+[Payara documentation](https://docs.payara.fish/community/docs/5.2020.6/documentation/user-guides/upgrade-payara.html)
+
+### New Banner API, Obsolete DB Settings
+
+The functionality previously provided by the DB settings :StatusMessageHeader and ::StatusMessageText is no longer supported and is now provided through the Manage Banner Messages API. Learn more in the [API Guide](https://guides.dataverse.org/en/5.3/api/).
+
+### New Database Settings and JVM Options
+
+Several new JVM options have been added in this release:
+
+- dataverse.db.name
+- dataverse.db.user
+- dataverse.db.password
+- dataverse.db.host
+- dataverse.db.port
+
+For an overview of these new options, please see the
+[Installation Guide](https://guides.dataverse.org/en/5.3/installation/config.html#jvm-options)
+
+See above note about obsolete DB options.
+
+### Java Message System Configuration
+
+The Ingest process uses the Java Message System to create ingest tasks in a queue. That queue had been configured from command line or domain.xml before. This has now changed to being done
+in code.
+
+In the unlikely case you might want to change any of these settings, feel free to change and recompile or raise an issue on Github. See `IngestQueueProducer` for more details.
+
+If you want to clean up your existing installation, you can delete the old, unused queue like this:
+
+```shell
+asadmin delete-connector-connection-pool --cascade=true jms/IngestQueueConnectionFactoryPool
```

Comment:
for consistency, this should like the asadmin directions below, i.e:
- `<payara install path>/asadmin

Reply:
reformatted in https://github.com/IQSS/dataverse/commit/f82f970aa6e1706dcf631eebcb498c159f6ffc96

@scolapasta - should I add /bin here as well?

Reply:
yes, whoops meant to add that there...

Reply:
added in https://github.com/IQSS/dataverse/pull/7446/commits/faf0e2b13f70cd46a82a8a883f1e1cf2533d8d8f